test/game/components/chest_test.dart

```diff
@@ -1,13 +1,22 @@
 import 'package:flame_test/flame_test.dart';
 import 'package:flutter_test/flutter_test.dart';
+import 'package:mocktail/mocktail.dart';
 import 'package:very_good_adventures/game/game.dart';
 
+class MockInventoryBloc extends Mock implements InventoryBloc {}
+
+class MockPlayerBloc extends Mock implements PlayerBloc {}
+
 void main() {
   TestWidgetsFlutterBinding.ensureInitialized();
 
   group('Chest', () {
-    FlameTester(VeryGoodAdventuresGame.new).test('loads correctly',
-        (game) async {
+    FlameTester(() {
+      return VeryGoodAdventuresGame(
+        playerBloc: PlayerBloc(),
```

Comment:
shouldn't we be using a MockPlayerBloc here as well?

Reply:
Ooops, good catch!

```suggestion
        playerBloc: MockPlayerBloc(),
```